agent/wptdriver/web_browser.cc

```diff
@@ -232,7 +232,13 @@ bool WebBrowser::RunAndWait() {
           ret = true;
           _status.Set(_T("Waiting up to %d seconds for the test to complete"), 
                       (_test._test_timeout / SECONDS_TO_MS) * 2);
-          DWORD wait_time = _test._test_timeout * 2;
+
+		  DWORD wait_time;
+		  if (_test._enable_per_test_timeout && _test._timeout_value_in_seconds != 0)
+			  wait_time = _test._timeout_value_in_seconds * SECONDS_TO_MS; //Strict limit on this value

```

Comment:
It was basically a large amount of leeway. It was added to timeout value that was specified in config.ini in the WPT folder. I kept it in if they weren't using our new timeout code. Otherwise, our new timeout code will be very specific and give them STRICTLY the amount of time they provisioned.

Later on in the code there is a 10 second delay anyways to allow the browser to close, so they have a bit of lee-way anyways for the wpt hook to finish before actually closing the browser.